tests/release_notes/test_record_factory.py

```
@@ -300,22 +315,29 @@ def test_generate_with_no_issues(mocker, request):
+    assert 2 == len(records)
+
     assert isinstance(records[101], PullRequestRecord)
     assert isinstance(records[102], PullRequestRecord)
 
+    rec_pr1 = cast(PullRequestRecord, records[101])
+    rec_pr2 = cast(PullRequestRecord, records[102])
+
     # Verify that PRs are registered
-    assert 1 == records[101].pulls_count
-    assert 1 == records[102].pulls_count
+    assert 0 == rec_pr1.issues_count()
```

Ruff (0.12.2)

Yoda condition detected

Rewrite as `rec_pr1.issues_count() == 0`

(SIM300)